plone/synchronize/tests.py

```diff
@@ -38,15 +38,8 @@ def test_instance_method(self):
 
         self.assertEqual("one", item)
 
-        # raises exception
-
-        raised = False
-        try:
+        with self.assertRaises(IndexError):
             shared_stack.pop()
```

Comment:
Excellent! As an almost 100% pytest user I did not know context managers are available for the unittest module.

Thanks, I learned something new!